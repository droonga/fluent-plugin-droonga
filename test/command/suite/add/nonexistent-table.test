{
  "type": "add",
  "dataset": "Droonga",
  "body": {
    "table": "NonExistent",
    "key": "key",
    "values": {
    }
  }
}
