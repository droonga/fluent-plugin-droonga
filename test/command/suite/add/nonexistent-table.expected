[
  "droonga.message",
  0,
  {
    "inReplyTo": null,
    "statusCode": 404,
    "type": "add.result",
    "body": {
      "name": "TableNotFound",
      "message": "Table <NonExistent> is not found"
    }
  }
]
